scripts/openbabel-python.i

```diff
@@ -280,6 +280,13 @@ namespace std { class stringbuf {}; }
 %include <openbabel/format.h>
 %include <openbabel/obconversion.h>
 %include <openbabel/obfunctions.h>
+
+//avoid conflicts with OBElement
+%rename(resC) OpenBabel::OBResidueIndex::C;
+%rename(resI) OpenBabel::OBResidueIndex::I;
+%rename(resU) OpenBabel::OBResidueIndex::U;
+
```

Comment:
This isn't the way I'd fix this, but I'll send in a pull request myself as this has been delayed enough.

Reply:
How do you suggest fixing this?  This was the best solution I could come up with given the flattened namespace, but I agree it is unappealing.

Reply:
I would wrap them (i.e. the elements) in #ifndef SWIG, and redefine them 'natively' in each of the bindings. We shouldn't be crossing over to C++ to access the values of constants in the first place. In general, we do this because we're lazy and they would easily go out of synch. Not so for elements->atomic numbers, they are not arbitrary enum values that may change at a future date. For Python, I was thinking of something like a class OBElements with attributes Carbon, etc. OEChem redefines enums/constants as OBElements_Carbon in the various bindings, but I don't like that very much.